src/tests/EventDetails.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```

Comment:
Good use of mock() and of dummy data here.  As with the App tests, it would be good to see:
1) slightly more steps of granular development, not just the main tests for successfully getting it on the page.  These could test similar factors to the factors of the EventDetails that could be tested in the App test.  These would be the unit versions rather than the integration tests for this component present in the app tests.
2) error handling and failure state tests.  It's important to keep in mind what can go wrong and anticipate it, building proper handling into our app in the process.